### ISSUE TITLE
studio(a11y): toolbar options have no focus state

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#536]
## Current behaviour

When tabbing through the content with keyboard in editor mode (https://deckdeckgo.com/editor/[id]), I noticed that none of the options in the toolbar have a **visible** focus state. I can't see where I currently am in the page.

![20191214-deckdeckgo-editor](https://user-images.githubusercontent.com/1763537/70848771-91d74580-1e76-11ea-96d7-7eb01e06f41c.gif)
_Reproducing issue (after second tab no visual clue)_

## Expected behaviour

To have a clue what option is selected when tabbing through with keyboard only navigation.

**Comment by peterpeterparker** [comment]
I might have to have a general look "tab" usage in the editor. If my deck contains more than one slide and if I tab in the last element of one of them, then the next slide receive the focus but the "swipe" doesn't occurs. Not good.

**Comment by peterpeterparker** [comment]
This issue will be solved in our next version @donroyco

If I would "tab" from an element on Slide X to an element on Slide X+1 or X-1, then we will notice this, fix the automatic scroll applied by the browser and swipe automatically to the related slide.
